### PR TITLE
Text Overflow in People Section

### LINKED ISSUE #7333: Text Overflow in People section
In https://publiclab.org/people Text for some people is more than div size.
Refer to following Image
Screenshot from 2020-01-23 17-33-45

### PR DESCRIPTION
**Pull request by urvashigupta7**
Fixes #7333
This will resolve the text overflow in people section.
Screenshot from 2020-01-27 20-28-19
-> See the textoverflow in user card "techlearningcollective"

### CODE REVIEW

**Comment by IshaGupta18** [comment]
Hey @urvashigupta7 could you please rename your PR's title and description more clearly about what it is solving?

**Comment by urvashigupta7** [comment]
### After the change

Screenshot from 2020-01-27 20-20-28

It will be fully responsive as well

**Comment by urvashigupta7** [comment]
@

> Hey @urvashigupta7 could you please rename your PR's title and description more clearly about what it is solving?

Done! Please have a look

**Comment by urvashigupta7** [comment]
@pydevsg The mobile version
Screenshot from 2020-01-27 21-42-29

File: Gemfile.lock [hunk 1]
```diff
@@ -662,4 +662,4 @@ RUBY VERSION
    ruby 2.4.4p296
 
 BUNDLED WITH
-   1.17.3
+   2.1.4
```
Comment (cesswairimu) [inline Gemfile.lock]: @urvashigupta7, could be due to the change of bundler version...could you please set it back to `1.17.3`
Thanks

**Comment by urvashigupta7** [comment]
@cesswairimu It is still failing. Please check

**Comment by urvashigupta7** [comment]
@cesswairimu Done, Passed all checks

**Comment by cesswairimu** [comment]
Great all test passing :rocket:

**Comment by SidharthBansal** [comment]
@cesswairimu can we merge this?

Great work @urvashigupta7

**Comment by urvashigupta7** [comment]
Thanks @SidharthBansal 😊

**Comment by cesswairimu** [comment]
Yeah, I think we can, one last thing...I believe there was a map on this page? Is it still there?

**Comment by urvashigupta7** [comment]
@cesswairimu Yes

**Comment by cesswairimu** [comment]
great we are ready to merge now

**Comment by urvashigupta7** [comment]
@cesswairimu After observing the changes in https://stable.publiclab.org/ I think I need to make one small change which is  user card height equal for all. This wasn't visible before. 
Screenshot_20200131_182322_com android chrome

**Comment by cesswairimu** [comment]
great catch, just create another issue and assign it to yourself. Thanks for following up

**Comment by urvashigupta7** [comment]
Okay

**Comment by urvashigupta7** [comment]
@cesswairimu I don't think I have permissions to assign task to myself. Can you do it?

**Comment by urvashigupta7** [comment]
I have created the issue.

**Comment by cesswairimu** [comment]
No worries assigning it to you

**Comment by urvashigupta7** [comment]
Thanks

**Comment by SidharthBansal** [comment]
Awesome collab

**Pull request merged**